README.md

```diff
@@ -3,25 +3,26 @@
 The F3 Peak City website site is built using Hugo and a fork of the [Beautiful Hugo theme](https://github.com/f3peakcity/beautifulhugo).
 
 The site is published using GitHub pages, available at the following URLs:
-* Staging: [https://f3peakcity.github.io/](https://f3peakcity.github.io/)
```

Comment:
Just curious, will this URL still work be up as a staging one that reflects `main` but isn't necessarily promoted to the `.com` address like it is now?

Reply:
Unfortunately, not. If we switch to netlify we could do something clever like that. The issue is one becomes an alias of the other to identical.